src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -267,7 +269,7 @@ internal void PublishWorkerProcessReadyEvent(FunctionEnvironmentReloadResponse r
             _eventManager.Publish(wpEvent);
         }
 
-        internal void PublishRpcChannelReadyEvent(RpcEvent initEvent)
+        internal void SendDownloadDependencyRequest(RpcEvent initEvent)
```

Comment:
With the refactoring, the fact that the channel ready event is sent from this method isn't clear (ideally this wouldn't be responsible for both messages, given the name). Can we refactor this to add clarity?

Reply:
Changing the name of the method to SendDownloadDependencyRequestOrPublishRpcChannelReadyEvent for clarity